system-tests/maven-external-upload-test/build.gradle.kts

```diff
@@ -3,8 +3,6 @@ plugins {
 }
 
 tasks.test {
-	environment("AGENT_VERSION", rootProject.extra["appVersion"].toString())
```

Comment:
There was an inconsistency between the version numbers being used (with and without -SNAPSHOT), which led to testing the wrong version of the agent/plugin etc. so I centralized that part into `buildSrc/src/main/kotlin/com.teamscale.system-test-convention.gradle.kts`